### ISSUE TITLE
[Failing test] gce-cos-master-alpha-features (ci-kubernetes-e2e-gci-gce-alpha-features)

### ISSUE THREAD

**Issue by alenkacz** [kubernetes/kubernetes#85196]
**Which jobs are failing**:
gce-cos-master-alpha-features
conformance-GCE-master
skew-cluster-latest-kubectl-stable1-gce
gci-gce-ingress
gce-cos-master-scalability-100

**Which test(s) are failing**:
test start up

**Since when has it been failing**:
12th November 23:16 PST

**Testgrid link**:
https://testgrid.k8s.io/sig-release-master-blocking#gce-cos-master-alpha-features

**Reason for failure**:

```
W1113 09:05:35.030] Failed to execute 'curl -s  http://127.0.0.1:4002/v2/members/$(curl -s  http://127.0.0.1:4002/v2/members -XGET | sed 's/{\"id/\n/g' | grep bootstrap-e2e-master\" | cut -f 3 -d \") -XDELETE -L 2>/dev/null' on bootstrap-e2e-master despite 5 attempts
W1113 09:05:35.030] Last attempt failed with: 
W1113 09:05:39.900] Updated [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/zones/us-west1-b/instances/bootstrap-e2e-master].
W1113 09:06:44.042] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/zones/us-west1-b/instances/bootstrap-e2e-master].
W1113 09:07:00.168] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/firewalls/bootstrap-e2e-master-https].
W1113 09:07:02.255] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/firewalls/bootstrap-e2e-minion-all].
W1113 09:07:06.404] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/firewalls/bootstrap-e2e-master-etcd].
W1113 09:07:12.867] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/regions/us-west1/addresses/bootstrap-e2e-master-ip].
W1113 09:07:31.322] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/firewalls/bootstrap-e2e-default-internal-master].
W1113 09:07:32.079] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/firewalls/bootstrap-e2e-default-internal-node].
W1113 09:07:33.401] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/firewalls/bootstrap-e2e-default-ssh].
I1113 09:07:34.497] Deleting firewall rules remaining in network bootstrap-e2e: 
W1113 09:08:30.754] Deleted [https://www.googleapis.com/compute/v1/projects/k8s-boskos-gce-project-16/global/networks/bootstrap-e2e].
W1113 09:08:31.142] W1113 09:08:31.140186    9616 loader.go:223] Config not found: /workspace/.kube/config
I1113 09:08:31.407] Property "clusters.k8s-boskos-gce-project-16_bootstrap-e2e" unset.
W1113 09:08:31.509] W1113 09:08:31.407002    9663 loader.go:223] Config not found: /workspace/.kube/config
W1113 09:08:31.509] W1113 09:08:31.407293    9663 loader.go:223] Config not found: /workspace/.kube/config
W1113 09:08:31.645] W1113 09:08:31.645324    9710 loader.go:223] Config not found: /workspace/.kube/config
W1113 09:08:31.646] W1113 09:08:31.646206    9710 loader.go:223] Config not found: /workspace/.kube/config
I1113 09:08:31.747] Property "users.k8s-boskos-gce-project-16_bootstrap-e2e" unset.
W1113 09:08:31.856] W1113 09:08:31.856358    9758 loader.go:223] Config not found: /workspace/.kube/config
W1113 09:08:31.857] W1113 09:08:31.857260    9758 loader.go:223] Config not found: /workspace/.kube/config
I1113 09:08:31.959] Property "users.k8s-boskos-gce-project-16_bootstrap-e2e-basic-auth" unset.
I1113 09:08:32.099] Property "contexts.k8s-boskos-gce-project-16_bootstrap-e2e" unset.
I1113 09:08:32.102] Cleared config for k8s-boskos-gce-project-16_bootstrap-e2e from /workspace/.kube/config
I1113 09:08:32.103] Done
W1113 09:08:32.203] W1113 09:08:32.097713    9804 loader.go:223] Config not found: /workspace/.kube/config
W1113 09:08:32.204] W1113 09:08:32.098096    9804 loader.go:223] Config not found: /workspace/.kube/config
W1113 09:08:32.204] 2019/11/13 09:08:32 process.go:155: Step './hack/e2e-internal/e2e-down.sh' finished in 7m13.536227968s
W1113 09:08:32.205] 2019/11/13 09:08:32 process.go:96: Saved XML output to /workspace/_artifacts/junit_runner.xml.
W1113 09:08:37.490] 2019/11/13 09:08:37 main.go:319: Something went wrong: starting e2e cluster: error during ./hack/e2e-internal/e2e-up.sh: exit status 2
W1113 09:08:37.490] Traceback (most recent call last):
W1113 09:08:37.491]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 778, in <module>
W1113 09:08:37.493]     main(parse_args())
W1113 09:08:37.493]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 626, in main
W1113 09:08:37.493]     mode.start(runner_args)
W1113 09:08:37.494]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 262, in start
W1113 09:08:37.494]     check_env(env, self.command, *args)
W1113 09:08:37.494]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 111, in check_env
W1113 09:08:37.494]     subprocess.check_call(cmd, env=env)
W1113 09:08:37.494]   File "/usr/lib/python2.7/subprocess.py", line 190, in check_call
W1113 09:08:37.495]     raise CalledProcessError(retcode, cmd)
W1113 09:08:37.496] subprocess.CalledProcessError: Command '('kubetest', '--dump=/workspace/_artifacts', '--gcp-service-account=/etc/service-account/service-account.json', '--up', '--down', '--test', '--provider=gce', '--cluster=bootstrap-e2e', '--gcp-network=bootstrap-e2e', '--check-leaked-resources', '--extract=ci/latest', '--gcp-node-image=gci', '--gcp-zone=us-west1-b', '--runtime-config=api/all=true', '--test_args=--ginkgo.focus=\\[Feature:(Audit|PodPreset|ExpandCSIVolumes|ExpandInUseVolumes|RunAsGroup|TTLAfterFinished|NodeLease|VolumeSnapshotDataSource|CSIInlineVolume)\\]|Networking --ginkgo.skip=Networking-Performance|IPv6|Feature:Volumes|csi-hostpath-v0 --minStartupPods=8', '--timeout=180m')' returned non-zero exit status 1
E1113 09:08:37.505] Command failed
```

**Anything else we need to know**:
/sig testing
/milestone v1.17

/priority critical-urgent

/cc @droslean  @hasheddan @alejandrox1

**Comment by liggitt** [comment]
broken by https://github.com/kubernetes/kubernetes/pull/76586

```
Nov 13 07:19:59.610320 bootstrap-e2e-master configure.sh[733]: mv: cannot stat '/home/kubernetes/kubernetes/LICENSE': No such file or directory
Nov 13 07:19:59.610997 bootstrap-e2e-master systemd[1]: kube-master-installation.service: Main process exited, code=exited, status=1/FAILURE
Nov 13 07:19:59.611147 bootstrap-e2e-master systemd[1]: kube-master-installation.service: Failed with result 'exit-code'.
Nov 13 07:19:59.611795 bootstrap-e2e-master systemd[1]: Failed to start Download and install k8s binaries and configurations.
```

**Comment by liggitt** [comment]
root cause is #76093
fix in #85210

**Comment by liggitt** [comment]
revert merged, job is green

/close
